azure-pipelines-ci.yml

```diff
@@ -34,14 +34,24 @@ jobs:
     workingDirectory: $(Build.ArtifactStagingDirectory)
   - script: $(Build.ArtifactStagingDirectory)\windows-dependencies\vcpkg.exe integrate install
     displayName: 'Install dependencies'
+  - bash: |
+      set -ex
+      curl -L https://binaries.openttd.org/extra/opengfx/0.5.2/opengfx-0.5.2-all.zip > opengfx-0.5.2-all.zip
+      unzip opengfx-0.5.2-all.zip
+      rm -f opengfx-0.5.2-all.zip
+    displayName: 'Install OpenGFX'
+    workingDirectory: '$(Build.SourcesDirectory)/bin/baseset'
```

Comment:
Depending on how releases eventually get done, this might cause issues/complications with packaging?

Reply:
install.nsi doesn't pick tar files so it should be ok